### DISCUSSION TITLE
ListItem textAlign not passed through to inner text component

### DISCUSSION THREAD

**Discussion by jmatsushita** [tamagui/tamagui#1378] [General]
### Current Behavior

Using `textAlign="right"` in a `ListItem` component has no effect.

### Expected Behavior

The behavior described in the docs:
> Passes "textAlign" down to the inner text component

### Tamagui Version

```markdown
1.36.6
```

### Reproduction

This code doesn't pass textAlign through to the `<Title>` and `<SubTitle>` components

https://github.com/tamagui/tamagui/blob/7e837efbfbd0edbdc0946773b95d82906b58e081/packages/list-item/src/ListItem.tsx#L244-L260

**Comment by natew** [comment]
This is sort of a feature request, but also if you need a lot of customization you probably should just roll your own. Eventually we'll make this a composable component API.
